api/v1/runtime_types.go

```diff
@@ -46,8 +46,6 @@ type CBContainersRuntimeSensorSpec struct {
 	Probes CBContainersFileProbesSpec `json:"probes,omitempty"`
 	// +kubebuilder:default:={port: 7071}
 	Prometheus CBContainersPrometheusSpec `json:"prometheus,omitempty"`
-	// +kubebuilder:default:=2
-	VerbosityLevel *int `json:"verbosity_level,omitempty"`
```

Comment:
In general, dropping a field from here is a breaking API change. That's because the API server verifies the fields of the schema.
What we can do in general is to leave the field here and ignore it, or show a warning if it was filled in by the user.